pkgs/by-name/pl/plasticscm-client-gui/package.nix

```diff
@@ -0,0 +1,102 @@
+{
+  buildFHSEnv,
+  extraLibs ? _: [ ],
+  extraPkgs ? _: [ ],
+  imagemagick,
+  lib,
+  makeWrapper,
+  plasticscm-client-core-unwrapped,
+  plasticscm-client-gui-unwrapped,
+  plasticscm-theme,
+}:
+assert plasticscm-client-gui-unwrapped.version == plasticscm-client-core-unwrapped.version;
+assert plasticscm-client-gui-unwrapped.version == plasticscm-theme.version;
```

Comment:
```suggestion
```
That just makes updates harder.

Reply:
We should instead write an update script which bumps both at the same time. Currently it is impossible to merge 2 PRs at the same time from r-ryantm which update the package, as that always hard fails.

Reply:
Ok, I've included an update script and removed the asserts.